apps/configuration/main.go

```diff
@@ -83,6 +108,13 @@ func main() {
 
 	dirs := strings.Split(*templatesDirs, ",")
 
+	switch config.Spec {
```

Comment:
there is no point adding `spec` to configuration app.

This change is breaking for us and for axway.

I think these configs can coexist without the need for `spec`

Reply:
done